verifier/rest/rest.go

```diff
@@ -89,7 +95,7 @@ func NewClient(ctx context.Context, projectID string, region string, opts ...opt
 }
 
 type restClient struct {
-	v1Client *v1.Client
+	v1Client *retryableClient
```

Comment:
Why can't we do the retry in the `restClient`'s functions?

The retry behavior is specific to the rest client, and then we don't need to create a wrapper around v1.Client anymore

Reply:
+1 but even deeper, I would like to do the retries within the actual client provided by the API library generation. See my other comment.

Its currently unsupported tho, so we would need to open a feature request and re-gen the libraries, if they even agree to do the work at all,.

Reply:
Done. We manipulated the CallOptions directly in the returned client.